front/components/assistant_builder/ActionScreen.tsx

```diff
@@ -554,6 +554,18 @@ export default function ActionScreen({
             it on the tables selected and use the results to generate an answer.
           </div>
 
+          <div>
```

Comment:
@Duncid feedback, no need for a carriage return. And let's use the `<Hoverable>` component to style the link.

Reply:
Thanks, updating

Reply:
Done